src/option.h

```diff
@@ -721,4 +721,32 @@ namespace librealsense
    private:
        std::atomic<bool>   _is_active;
    };
+
+   class ds5_thermal_handler;
+   class thermal_compensation : public option
```

Comment:
I think maybe this class needs to go in a ds5-specific header?

Reply:
Done